can/message.py

```diff
@@ -140,7 +141,13 @@ def __eq__(self, other):
                 self.is_remote_frame == other.is_remote_frame and
                 self.is_error_frame == other.is_error_frame and
                 self.is_fd == other.is_fd and
-                self.bitrate_switch == other.bitrate_switch)
+                self.bitrate_switch == other.bitrate_switch
+            )
+        else:
+            raise NotImplementedError()
```

Comment:
Shouldn’t it be just False?

Reply:
> A rich comparison method may return the singleton `NotImplemented` if it does not implement the operation for a given pair of arguments.

Do we support it or is it just False? Maybe, the other object would be able to compare a message to them?

Reply:
Still, we should return the singleton and not raise the Error if we decide to leave it like this.

Reply:
According to [this](https://eev.ee/blog/2012/03/24/python-faq-equality/#overloading), when we return `NotImplemented`, the other object "`b`" is asked if it can compare itself to the message. I guess that's desirable, or at least not harming.

Reply:
Makes sense!